Plugins/Flow.Launcher.Plugin.BrowserBookmark/FirefoxBookmarkLoader.cs

```diff
@@ -9,10 +9,10 @@ namespace Flow.Launcher.Plugin.BrowserBookmark
 {
     public class FirefoxBookmarkLoader : IBookmarkLoader
     {
-        private const string queryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
+        private const string QueryAllBookmarks = @"SELECT moz_places.url, moz_bookmarks.title
```

Comment:
this is private variable, why change to uppercase?

Reply:
updated.

Reply:
Well because it is a const string🤣 keep lowercase is fine as well